oml/functional/metrics.py

```diff
@@ -464,13 +520,10 @@ def calc_fnmr_at_fmr(pos_dist: Tensor, neg_dist: Tensor, fmr_vals: Tuple[float,
     return fnmr_at_fmr
 
 
-def calc_fnmr_at_fmr_from_matrices(
-    distance_matrix: FloatTensor, mask_gt: BoolTensor, fmr_vals: Tuple[float, ...]
-) -> TMetricsDict:
+def calc_fnmr_at_fmr_(pos_dist: FloatTensor, neg_dist: FloatTensor, fmr_vals: Tuple[float, ...]) -> TMetricsDict:
```

Comment:
Maybe something more meaningful than just a postfix `_` to the main `calc_fnmr_at_fmr` function? 😁

Reply:
okay
changed to `calc_fnmr_at_fmr_by_distances`